uaclient/tests/test_security_status.py

```diff
@@ -28,11 +28,14 @@ def mock_origin(
 
 
 def mock_version(
-    version: str, origin_list: List[mock.MagicMock] = []
+    version: str,
+    size: int,
```

Comment:
Just as a quick suggestion, you might set a default of `1` for this and only change where needed (:
Just as a quick comment, oh my god those mocks :face_exhaling:

Reply:
good idea! I'll do that

Reply:
updated!